apps/web/app/layout.tsx

```diff
@@ -74,46 +74,6 @@ export default async function RootLayout({ children }: { children: React.ReactNo
             }}
           />
         )}
-        <script
-          nonce={nonce}
-          id="headScript"
-          // eslint-disable-next-line react/no-danger
-          dangerouslySetInnerHTML={{
-            __html: `
-              window.calNewLocale = "${locale}";
-              (function applyTheme() {
-                try {
-                  const appTheme = localStorage.getItem('app-theme');
-                  if (!appTheme) return;
-
-                  let bookingTheme, username;
-                  for (let i = 0; i < localStorage.length; i++) {
-                    const key = localStorage.key(i);
-                    if (key.startsWith('booking-theme:')) {
-                      bookingTheme = localStorage.getItem(key);
-                      username = key.split("booking-theme:")[1];
```

Comment:
This code shouldn't be needed. next-theme automatically installs a JS snippet on the page that manages the light/dark classes.

Plus, this code completely ignores embeds. It is better handled through getThemeProviderProps